Formula/rtx.rb

```diff
@@ -18,6 +18,11 @@ class Rtx < Formula
 
   depends_on "rust" => :build
 
+  on_linux do
+    depends_on "pkg-config" => :build
```

Comment:
you sure this is required? sccache doesn't have it

Reply:
it does however set this which I was checking to see if was required or not:

```
    ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix if OS.linux?
```

Reply:
looks like the build passed without it

Reply:
> you sure this is required? sccache doesn't have it

it does need it, it was actually alerting in this run, https://github.com/Homebrew/homebrew-core/actions/runs/5706888197/job/15464877308

```
    run pkg_config fail: Could not run `PKG_CONFIG_ALLOW_SYSTEM_CFLAGS="1" "pkg-config" "--libs" "--cflags" "openssl"`
    The pkg-config command could not be found.
 ```

Reply:
> it does however set this which I was checking to see if was required or not:
> 
> ```
>     ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix if OS.linux?
> ```

I think it is fine.

Reply:
in the build log, I can see the linkage is done correctly
<img width="699" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/35f6a04a-1786-4bc8-b204-381a3e3b5340">